pkg/tests/tasks/security/certmanager/yaml/cert-manager-operator.yaml

```diff
@@ -18,4 +18,4 @@ spec:
   name: openshift-cert-manager-operator
   source: redhat-operators
   sourceNamespace: openshift-marketplace
-  startingCSV: cert-manager-operator.v1.10.2
+  startingCSV: cert-manager-operator.v1.11.1
```

Comment:
Will it work on OCP 4.10-4.11? Don't we need solution like we have for metallb?
- https://github.com/maistra/maistra-test-tool/blob/main/pkg/metallb/yaml/metallb-operator.yaml#L27
- https://github.com/maistra/maistra-test-tool/blob/main/pkg/metallb/install.go#L25

Reply:
We only execute this test cases related to cert manager on SMCP 2.4+ and OCP 4.12+:
```
if smcpVer.LessThan(version.SMCP_2_4) {
			t.Skip("istio-csr is not supported in SMCP older than v2.4")
		}
		ocpVersion := version.ParseVersion(oc.GetOCPVersion(t))
		if ocpVersion.LessThan(version.OCP_4_12) {
			t.Skip("istio-csr is not supported in OCP older than v4.12")
		}
```
We should be supporting other OCP versions?

Reply:
Yes, I think so. I don't see any reason to not support it on 4.10 and 4.11.

Reply:
Ok, @jewertow I'm going to open a new ticket to manage that because this one is only related to the failure. wdyt? it's ok for you? I think worth to open a new one because we are not executing the test now against those versions, so we are not decreasing the coverage

Reply:
This is the Jira ticket to address that change: https://issues.redhat.com/browse/OSSM-4257